scripts/build_image

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# set readable variables
+export JAVA_VERSION_TEMPLATE="$1"
+export NODE_VERSION_TEMPLATE="$2"
+export MBT_VERSION="$3"
+
+# print java and node version parameter
+echo "Java version template: ${JAVA_VERSION_TEMPLATE}, Node version template: ${NODE_VERSION_TEMPLATE}"
+
+# validate java and node major version
+export JAVA_MAJOR_VERSION="${JAVA_VERSION_TEMPLATE}"
+export NODE_MAJOR_VERSION="$(echo ${NODE_VERSION_TEMPLATE} |awk -F. '{printf "%d", $1}')"
+
+echo "Java major version: ${JAVA_MAJOR_VERSION}, Node major version: ${NODE_MAJOR_VERSION}"
+
+if ([ "$JAVA_MAJOR_VERSION" -ne "8" ] && [ "$JAVA_MAJOR_VERSION" -ne "11" ] && [ "$JAVA_MAJOR_VERSION" -ne "17" ]) || \
```

Comment:
I suspect this combination checking logic won't scale in an easy to understand 
manner if/when you will support many versions/combinations.

Reply:
maybe simplify the code in the future